core/modules/file.py

```diff
@@ -239,6 +239,35 @@ class File(Tree):
 
 	blobCacheTime = 60 * 60 * 24  # Requests to file/download will be served with cache-control: public, max-age=blobCacheTime if set
 
+	def write(self, filename: str, content: Any, mimetype: str = "text/plain"):
```

Comment:
For the sake of completeness add the return type.
```suggestion
	def write(self, filename: str, content: Any, mimetype: str = "text/plain") -> db.Key:
```

Reply:
```suggestion
	def write(self, filename: str, content: Any, mimetype: str = "text/plain", width: int = 0, height: int = 0) -> db.Key:
```